### PR TITLE
SDET-281: Create contrast-service sidecar docker container

### PR DESCRIPTION
**Pull request by fushi**

### CODE REVIEW

File: contrast/contrast-service.dockerfile [hunk 1]
```diff
@@ -0,0 +1,12 @@
+FROM buildpack-deps:stretch
```
Comment (gibbsjosh) [inline contrast/contrast-service.dockerfile:1]: I was curious if stretch was the common base image for utils, and it looks like there's no real standard across the dockerfiles in the repo. 👍   on going with one with a good tradeoff for image size versus extra libs needed.

**Review by gibbsjosh — APPROVED** [review]
👍  lgtm

File: contrast/contrast-service.dockerfile (hunk 1 above)
Comment (fushi) [reply contrast/contrast-service.dockerfile:1]: Yeah, we could alter this in the future, but stretch was the base for the python docker files I was working on at the time, so I used stretch.

File: contrast/contrast-service.dockerfile [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+run apt-get update && apt-get install apt-transport-https
+run curl https://contrastsecurity.jfrog.io/contrastsecurity/api/gpg/key/public | apt-key add -
+run echo "deb https://contrastsecurity.jfrog.io/contrastsecurity/debian-public/ stretch contrast" | tee /etc/apt/sources.list.d/contrastc.list
+run apt-get update && apt-get install contrast-service
+
+COPY ./contrast_security.yaml /etc/contrast/contrast_security.yaml
+
+run mv /etc/contrast/webserver/contrast_security.yaml /tmp
```
[the remark this answers is not in the document]
Comment (fushi) [reply contrast/contrast-service.dockerfile:10]: I could `&&` it, but I left it separate for clarity

**Pull request merged**